.github/workflows/CI.yaml

```diff
@@ -16,44 +16,19 @@ jobs:
             node: 18.x
             script: npm run test-nolint
 
-          - name: "Linux: Node 16"
-            # Includes 'firefox', 'chromium', and more.
-            # https://github.com/actions/virtual-environments/blob/ubuntu20/20220410.2/images/linux/Ubuntu2004-Readme.md
-            os: ubuntu-20.04
-            node: 16.x
-            # Run the lint step only once because, in March 2022, GitHub began
-            # automatically extracting ESLint warnings from build logs and turning them into
-            # checks shown inline in PR diffs, but doesn't de-duplicate these.
-            script: npm test
```

Comment:
At least one needs to run `npm test` to completion. Can you update the Node 18 one above to be that?

Reply:
done, thanks!